eventhubs/data-plane/azure-eventhubs/src/test/java/com/microsoft/azure/eventhubs/connstrbuilder/TransportTypeTest.java

```diff
@@ -89,7 +91,8 @@ public void transportTypeAmqpWebSocketsCreatesConnectionWithPort443() throws Exc
     public void transportTypeAmqpWebSocketsWithProxyCreatesConnectionWithCorrectPorts() throws Exception {
         int proxyPort = 8899;
         ProxyServer proxyServer = ProxyServer.create("localhost", proxyPort);
-        proxyServer.start(throwable -> {});
+        proxyServer.start(throwable -> {
+        });
```

Comment:
This makes me cry. I would much rather accept the previous line of code, but I know it won't be easy to bend CheckStyle to our will here, so it's ok.